rct229/rulesets/ashrae9012019/ruleset_functions/are_all_terminal_types_VAV.py

```diff
@@ -0,0 +1,31 @@
+from rct229.schema.schema_enums import SchemaEnums
+from rct229.utils.utility_functions import find_exactly_one_terminal_unit
+
+TERMINAL_TYPE = SchemaEnums.schema_enums["TerminalOptions"]
+
+
+def are_all_terminal_types_VAV(rmd: dict, terminal_unit_id_list: list[str]) -> bool:
+    """
+    Returns TRUE if all of the terminal unit types input to this function are variable air volume (VAV) or are equal to null.
+    It returns FALSE if any of the terminal units are of a type other than variable air volume (VAV) or null.
+
+    Parameters
+    ----------
+    rmd: json,
```

Comment:
I would recommend changing the type hint for rmd in the docstring to dict to align with the function description.

Reply:
Addressed